pmd-cli/src/test/java/net/sourceforge/pmd/cli/TreeExportCliTest.java

```diff
@@ -44,9 +44,9 @@ void testReadFile() throws Exception {
         File file = newFileWithContents("(a(b))");
         final CliExecutionResult result = runCliSuccessfully("--file", file.getAbsolutePath(), "-f", "xml", "-PlineSeparator=LF");
         result.checkStdOut(equalTo("<?xml version='1.0' encoding='UTF-8' ?>\n"
-                                       + "<dummyRootNode Image=''>\n"
-                                       + "    <dummyNode Image='a'>\n"
-                                       + "        <dummyNode Image='b' />\n"
+                                       + "<dummyRootNode Image='' Lines='[, , ]'>\n"
```

Comment:
Can we put some double quotes around string values in sequences?

Reply:
I don't think so without doing some nasty stuff… this is produced in `net.sourceforge.pmd.lang.rule.xpath.Attribute` by simply doing a `toString()` on the attribute raw value (an `Object`). This is how `AbstractCollection.toString()` does it…

Reply:
Ok then let's not care